tensorflow_lite_support/cc/task/vision/landmark_detector.h

```diff
@@ -0,0 +1,120 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+#ifndef TENSORFLOW_LITE_SUPPORT_CC_TASK_VISION_LANDMARK_DETECTOR_H_
+#define TENSORFLOW_LITE_SUPPORT_CC_TASK_VISION_LANDMARK_DETECTOR_H_
+
+#include <memory>
+#include <vector>
+
+#include "absl/status/status.h"
+#include "tensorflow/lite/core/shims/cc/kernels/register.h"
+#include "tensorflow_lite_support/cc/port/integral_types.h"
+#include "tensorflow_lite_support/cc/port/statusor.h"
+#include "tensorflow_lite_support/cc/task/vision/core/base_vision_task_api.h"
+#include "tensorflow_lite_support/cc/task/vision/core/frame_buffer.h"
+#include "tensorflow_lite_support/cc/task/vision/proto/landmark_detector_options_proto_inc.h"
+#include "tensorflow_lite_support/cc/task/vision/proto/landmarks_proto_inc.h"
+
+namespace tflite {
+namespace task {
+namespace vision {
+
+// Performs landmark detection on images.
+//
+// The API expects a TFLite model with optional TFLite Model Metadata.
+//
+// Input tensor:
+//  (kTfLiteUInt8)
+//    - image input of size `[batch x height x width x channels]`.
+//    - batch inference is not supported (`batch` is required to be 1).
+//    - only RGB inputs are supported (`channels` is required to be 3).
+//    - `NormalizationOptions` is not required.

```

Comment:
``NormalizationOptions` is not required` is only valid for movenet. But LandmarkDetection can serve other models as long as they fit the input/output requirements. So just remove  `- `NormalizationOptions` is not required.`.

Please also update the readme file.